addons/mail/static/src/core/common/autoresize_input.js

```diff
@@ -50,9 +52,15 @@ export class AutoresizeInput extends Component {
                 this.inputRef.el.blur();
                 break;
             case "Escape":
+                markEventHandled(ev, "AutoresizeInput.discard");
```

Comment:
Why not stop propagation?
Marking event being handled is nice in rare cases, but here `.stopPropagation()` would lead to the same overall behaviour with less LOCs and code maintenance (no diff in `chat_window.js`).